src/bson/object_id.cr

```diff
@@ -48,7 +48,7 @@ class BSON
       t = LibBSON.bson_oid_get_time_t(@handle)
       ts = LibC::Timespec.new
       ts.tv_sec = t
-      Time.new(ts, Time::Kind::Utc)
+      Time.new(ts, location: Time::Location.load("utc"))
```

Comment:
Isn't it better to use `Time::Location::UTC` directly? real question I don't know, I just think that constant access looks faster than method call.